pom.xml

```diff
@@ -79,55 +127,73 @@
 			<groupId>javax.jcr</groupId>
 			<artifactId>jcr</artifactId>
 			<version>2.0</version>
-            <optional>true</optional>
+			<optional>true</optional>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
-            <!-- A recent version is required to use Sling mocks in tests, if that's a problem we'll need to change them -->
+			<!-- A recent version is required to use Sling mocks in tests, if that's a problem we'll need to change them -->
 			<groupId>org.apache.sling</groupId>
 			<artifactId>org.apache.sling.api</artifactId>
 			<version>2.3.0</version>
 			<optional>true</optional>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.sling</groupId>
 			<artifactId>org.apache.sling.commons.testing</artifactId>
 			<version>2.0.22</version>
 			<scope>test</scope>
 			<exclusions>
-			    <exclusion>
```

Comment:
Please revert this whitespace-only change, it makes checking the history more difficult.